### ISSUE TITLE
Error querying the Blockchain

### ISSUE THREAD

**Issue by Ottibalboa98** [particl/copay#53]
Hi!

I've created a new btc wallet within copay. I transferred my btc from my old copay wallet to the newly created one.

After this Transaction (which has already been confirmed) i get the following error message.
![Screenshot_2021-01-08-20-43-39-941_io particl copay](https://user-images.githubusercontent.com/77171504/104057724-ed4b4280-51f2-11eb-83b5-38b07bf8b56c.jpg)

Whats the deal behind that? It's frustrating that the Transaction gets somewhere lost!

How can this be fixed?

**Comment by wpeart** [comment]
Same here. And a friend as well. Just tried to import the wallet to BitPay, doesn’t work

**Comment by Ottibalboa98** [comment]
Did you also contact the admin per mail? I've found this email address on google play store: admin@particl.io

**Comment by wpeart** [comment]
> Did you also contact the admin per mail? I've found this email address on google play store: [admin@particl.io](mailto:admin@particl.io)

Thankfully I was able to import my particl wallet to flarewallet , thanks for the reply

**Comment by Ottibalboa98** [comment]
Good for you. How did you do that? I have tried to import the wallet to Exodus but i got an error message that the private key is invalid.  😠 Getting frustrated and still no reply from any developer either here or per mail. It's quite a lot if money i'm talking about and if i dont get any support soon i am considering taking legal action.

**Comment by wpeart** [comment]
> Good for you. How did you do that? I have tried to import the wallet to Exodus but i got an error message that the private key is invalid. 😠 Getting frustrated and still no reply from any developer either here or per mail. It's quite a lot if money i'm talking about and if i dont get any support soon i am considering taking legal action.

Yeah I know, it is quite frustrating. For me only flare wallet worked, just followed the steps to recover the wallet and it did work. Tried bitpay first with no success. Good luck mate.

**Comment by FRUSTRATDC** [comment]
I am having the same issue....where is my bitcoin? ERROR QUERYING THE BLOCKCHAIN. PLEAASE HELP

**Comment by justin420gue** [comment]
I am having the same issue....where is my bitcoin? ERROR QUERYING THE BLOCKCHAIN. PLEAASE HELP
![Screenshot_20210706-044530](https://user-images.githubusercontent.com/76177166/124570626-10a72100-de15-11eb-8b18-345aa0894299.png)

**Comment by WillPeart** [comment]
I'd recommend trying **Flare Wallet** is the only one that actually worked for me.

Good luck.

**Comment by Arkodark** [comment]
Hey Guys,

I have the same issue described here, so I downloaded Flare Wallet like advised and succeed to import my wallet but my balance show 0...
Any thoughs to solve this ?
Many thanks :)

**Comment by memo3g** [comment]
I have the same issue, i think this app is a scam, i restored my wallet to other app, and the result us $0 founds
